src/racing/memory.c

```diff
@@ -550,32 +386,6 @@ UNUSED u8* func_802A841C(u8* arg0, s32 arg1, s32 arg2) {
     return temp_v0;
 }
 
-u8* dma_textures(const char* texture, size_t arg1, size_t arg2) {
-    u8* temp_v0;
-    void* temp_a0;
-#ifdef TARGET_N64
-    temp_v0 = (u8*) gNextFreeMemoryAddress;
-#else
-
-    temp_v0 = (u8*) allocate_memory(arg2);
-#endif
-    temp_a0 = temp_v0 + arg2;
-    arg1 = ALIGN16(arg1);
-    arg2 = ALIGN16(arg2);
-#ifdef TARGET_N64
-    osInvalDCache((void*) temp_a0, arg1);
-    osPiStartDma(&gDmaIoMesg, 0, 0, (uintptr_t) &_other_texturesSegmentRomStart[SEGMENT_OFFSET(texture)],
-                 (void*) temp_a0, arg1, &gDmaMesgQueue);
-    osRecvMesg(&gDmaMesgQueue, &gMainReceivedMesg, (int) 1);
-    mio0decode((u8*) temp_a0, temp_v0);
-    gNextFreeMemoryAddress += arg2;
-#else
-    strncpy(temp_v0, texture, arg2);
-    // strcpy(temp_v0, texture);
-#endif
-    return temp_v0;
-}
-
 uintptr_t MIO0_0F(u8* arg0, uintptr_t arg1, uintptr_t arg2) {
     uintptr_t oldHeapEndPtr;
     void* temp_v0;
```

Comment:
and this

Reply:
prefer not remove it because mio0 would be helpful to get ghost save compatible with emulator